src/datastructures/Relationship.java

```diff
@@ -0,0 +1,71 @@
+package datastructures;
+
+import java.util.Objects;
+
+public class Relationship {
+	String name;
+	String class1;
+	String class2;
+	
+	//*********************************************************//
+	// Constructors //
+	//*********************************************************//
+	
+	public Relationship(String newname, String newclass1, String newclass2)
+	{
+		name = newname;
+		class1 = newclass1;
+		class2 = newclass2;
+	}
+	
+	//*********************************************************//
+	// Accessors & Mutators //
+	//*********************************************************//
+	
+	public String getName()
+	{
+		return name;
+	}
+	
+	public String getFirstClass()
+	{
+		return class1;
+	}
+	
+	public String getSecondClass()
+	{
+		return class2;
+	}
+	
+	public void setName(String newname)
+	{
+		name = newname;
+	}
+	
+	public void setFirstClass(String newclass1)
+	{
+		name = newclass1;
+	}
+
+	public void setSecondClass(String newclass2)
+	{
+		name = newclass2;
+	}
```

Comment:
I put these in just in case a relationship is mutable in the future, but we don't really ever set these outsides of the constructor at the moment.

Reply:
I actually just discovered something where we might need this. When we call renameClass (rename an entity), we also need to update any relationships associated with that entity. For example, suppose we have Entities E1 and E2, and a relationship between them called "Link." Let's say we then rename E2 to "EE." We need to then go and change the Relationship "Link" to link between E1 and EE since E2 doesn't exist anymore.

Reply:
Oh shoot you're right, definitely need to implement that :O